### PR TITLE
Olek31012019

### PR DESCRIPTION
**Pull request by ol3kjak**
could you

### CODE REVIEW

File: Git/Tasks/firstOne [hunk 1]
```diff
@@ -0,0 +1 @@
+text one
```
Comment (michalmietlinski) [inline Git/Tasks/firstOne]: Remove this file

File: Git/Tasks/secondOne [hunk 2]
```diff
@@ -0,0 +1 @@
+text 2
```
Comment (michalmietlinski) [inline Git/Tasks/secondOne]: Remove This file :P

File: Git/Tasks/text1.md [hunk 3]
```diff
@@ -0,0 +1 @@
+text one
```
Comment (michalmietlinski) [inline Git/Tasks/text1.md]: Remove This file :P

File: Git/Tasks/text2 [hunk 4]
```diff
@@ -0,0 +1 @@
+text 2
```
Comment (michalmietlinski) [inline Git/Tasks/text2]: Remove This file :P

**Review by michalmietlinski — CHANGES_REQUESTED** [review]

**Comment by ol3kjak** [comment]
2

**Pull request closed without merging**